app/scripts/controllers/rif/rns/register/index.js

```diff
@@ -180,4 +182,32 @@ export default class RnsRegister extends RnsJsDelegate {
     return `${dataSignature}${dataOwner}${dataSecret}${dataDuration}${dataName}`;
   }
 
+  /**
+   * Overrides the parent operation because rns-js is failing on this, we invoke the contract directly.
+   */
+  createSubdomain (domainName, subdomain, ownerAddress, parentOwnerAddress) {
+    domainName = this.addRskSuffix(domainName);
+    if (!ownerAddress) {
+      ownerAddress = this.address;
+    }
+    const node = namehash(domainName);
+    const label = web3Utils.sha3(subdomain);
+    const result = this.send(this.rnsContractInstance, 'setSubnodeOwner', [node, label, ownerAddress])
+    result.then(transactionReceipt => {
```

Comment:
We need to modify this in order to use transactionListener right? What is the idea? include another task for refactor?

Reply:
That code is old @marcosmartinez7, it's already using the listener now, i think github didn't refreshed the code for you. You can download this branch if you want to see